docs/elasticsearch.md

```diff
@@ -0,0 +1,49 @@
+## Elasticsearch
+
+Telescope uses the [Elasticsearch](https://www.elastic.co/what-is/elasticsearch) search engine for full-text indexing.
+We use the [@elastic/elasticsearch](https://www.npmjs.com/package/@elastic/elasticsearch)
+JavaScript module to interact with Elasticsearch in our code. In our tests, we use
+[Jest Manual Mocks](https://jestjs.io/docs/en/manual-mocks) to provide mocks
+for Elasticsearch.
+
+For information about installing Redis, see [our environment setup guide](environment-setup.md).
+
+## Elasticsearch Tips
+
+- Interact with Elasticsearch outside of Telescope using one of the [available clients](https://www.elastic.co/guide/en/elasticsearch/client/index.html).
+- You can also interact with Elasticsearch's exposed `REST API` using a browser or [cURL](https://curl.haxx.se/).
+- If you want to see the state of the cluster (`cURL`):
+
+```sh
+ curl -X GET 'localhost:9200/\_cluster/health?pretty'
```

Comment:
typo here, should be `/_cluster/` not `/\_cluster/`

Reply:
I saw you approved the PR. Should I change this or it's ok?

Reply:
Fixed